x/feerefunder/types/events.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+// feerefunder module event types
+const (
+	EventTypeDistributeAcknowledgementFee = "distribute_ack_fee"
+	EventTypeDistributeTimeoutFee         = "distribute_timeout_fee"
+	EventTypeLockFees                     = "distribute_lock_fees"
```

Comment:
**distribute**_lock_fees?
Seems like copy paste, you mean just `lock_fees`?

Reply:
my bad, fixed